Description: We ship the binary as exim4 instead of exim, fix manpage
 accordingly.
Author: Marc Haber <mh+debian-packages@zugschlus.de>, 
 Andreas Metzler <ametzler@bebt.de>
Last-Update: 2018-12-31
Forwarded: not-needed (upstream uses the "exim" name)

--- a/doc/exim.8
+++ b/doc/exim.8
@@ -1,9 +1,9 @@
-.TH EXIM 8
+.TH EXIM4 8
 .SH NAME
-exim \- a Mail Transfer Agent
+exim4 \- a Mail Transfer Agent
 .SH SYNOPSIS
 .nf
-.B exim [options] arguments ...
+.B exim4 [options] arguments ...
 .B mailq [options] arguments ...
 .B rsmtp [options] arguments ...
 .B rmail [options] arguments ...
@@ -40,7 +40,7 @@ local message on the standard input, wit
 recipients) is assumed. Thus, for example, if Exim is installed in
 \fI/usr/sbin\fP, you can send a message from the command line like this:
 .sp
-  /usr/sbin/exim -i <recipient-address(es)>
+  /usr/sbin/exim4 -i <recipient-address(es)>
   <message content, including all the header lines>
   CTRL-D
 .sp
@@ -125,8 +125,8 @@ ports, on multiple ports, and only on sp
 .sp
 When a listening daemon
 is started without the use of \fB\-oX\fP (that is, without overriding the normal
-configuration), it writes its process id to a file called exim\-daemon.pid
-in Exim's spool directory. This location can be overridden by setting
+configuration), it writes its process id to a file called
+/var/run/exim4/exim.pid. This location can be overridden by setting
 PID_FILE_PATH in Local/Makefile. The file is written while Exim is still
 running as root.
 .sp
@@ -180,7 +180,7 @@ available to admin users.
 This option operates like \fB\-be\fP except that it must be followed by the name
 of a file. For example:
 .sp
-  exim \-bem /tmp/testmessage
+  exim4 \-bem /tmp/testmessage
 .sp
 The file is read as a message (as if receiving a locally\-submitted non\-SMTP
 message) before any of the test expansions are done. Thus, message\-specific
@@ -206,7 +206,7 @@ If you want to test a system filter file
 can use both \fB\-bF\fP and \fB\-bf\fP on the same command, in order to test a system
 filter and a user filter in the same run. For example:
 .sp
-  exim \-bF /system/filter \-bf /user/filter </test/message
+  exim4 \-bF /system/filter \-bf /user/filter </test/message
 .sp
 This is helpful when the system filter adds header lines or sets filter
 variables that are used by the user filter.
@@ -258,8 +258,8 @@ This option runs a fake SMTP session as
 standard input and output. The IP address may include a port number at the end,
 after a full stop. For example:
 .sp
-  exim \-bh 10.9.8.7.1234
-  exim \-bh fe80::a00:20ff:fe86:a061.5678
+  exim4 \-bh 10.9.8.7.1234
+  exim4 \-bh fe80::a00:20ff:fe86:a061.5678
 .sp
 When an IPv6 address is given, it is converted into canonical form. In the case
 of the second example above, the value of \fI$sender_host_address\fP after
@@ -417,7 +417,7 @@ main configuration options to be written
 of one or more specific options can be requested by giving their names as
 arguments, for example:
 .sp
-  exim \-bP qualify_domain hold_domains
+  exim4 \-bP qualify_domain hold_domains
 .sp
 However, any option setting that is preceded by the word "hide" in the
 configuration file is not shown in full, except to an admin user. For other
@@ -445,7 +445,7 @@ written directly into the spool director
 .sp
 If \fB\-bP\fP is followed by a name preceded by +, for example,
 .sp
-  exim \-bP +local_domains
+  exim4 \-bP +local_domains
 .sp
 it searches for a matching named list of any type (domain, host, address, or
 local part) and outputs what it finds.
@@ -454,7 +454,7 @@ If one of the words \fBrouter\fP, \fBtra
 followed by the name of an appropriate driver instance, the option settings for
 that driver are output. For example:
 .sp
-  exim \-bP transport local_delivery
+  exim4 \-bP transport local_delivery
 .sp
 The generic driver options are output first, followed by the driver's private
 options. A list of the names of drivers of a particular type can be obtained by
@@ -539,7 +539,7 @@ This option is for testing retry rules,
 arguments. It causes Exim to look for a retry rule that matches the values
 and to write it to the standard output. For example:
 .sp
-  exim \-brt bach.comp.mus.example
+  exim4 \-brt bach.comp.mus.example
   Retry rule: *.comp.mus.example  F,2h,15m; F,4d,30m;
 .sp
  The first
@@ -552,7 +552,7 @@ rule is found that matches the host, one
 sought. Finally, an argument that is the name of a specific delivery error, as
 used in setting up retry rules, can be given. For example:
 .sp
-  exim \-brt haydn.comp.mus.example quota_3d
+  exim4 \-brt haydn.comp.mus.example quota_3d
   Retry rule: *@haydn.comp.mus.example quota_3d  F,1h,15m
 .TP 10
 \fB\-brw\fP
@@ -655,7 +655,7 @@ doing such tests.
 .TP 10
 \fB\-bV\fP
 This option causes Exim to write the current version number, compilation
-number, and compilation date of the \fIexim\fP binary to the standard output.
+number, and compilation date of the \fIexim4\fP binary to the standard output.
 It also lists the DBM library that is being used, the optional modules (such as
 specific lookup types), the drivers that are included in the binary, and the
 name of the runtime configuration file that is in use.
@@ -683,7 +683,7 @@ If no arguments are given, Exim runs in
 right angle bracket for addresses to be verified.
 .sp
 Unlike the \fB\-be\fP test option, you cannot arrange for Exim to use the
-readline() function, because it is running as \fIexim\fP and there are
+readline() function, because it is running as \fIexim4\fP and there are
 security issues.
 .sp
 Verification differs from address testing (the \fB\-bt\fP option) in that routers
@@ -796,14 +796,14 @@ command line item. \fB\-D\fP can be used
 string, in which case the equals sign is optional. These two commands are
 synonymous:
 .sp
-  exim \-DABC  ...
-  exim \-DABC= ...
+  exim4 \-DABC  ...
+  exim4 \-DABC= ...
 .sp
 To include spaces in a macro definition item, quotes must be used. If you use
 quotes, spaces are permitted around the macro name and the equals sign. For
 example:
 .sp
-  exim '\-D ABC = something' ...
+  exim4 '\-D ABC = something' ...
 .sp
 \fB\-D\fP may be repeated up to 10 times on a command line.
 Only macro names up to 22 letters long can be set.
@@ -938,8 +938,8 @@ never provoke a bounce. An empty sender
 string, or as a pair of angle brackets with nothing between them, as in these
 examples of shell commands:
 .sp
-  exim \-f '<>' user@domain
-  exim \-f "" user@domain
+  exim4 \-f '<>' user@domain
+  exim4 \-f "" user@domain
 .sp
 In addition, the use of \fB\-f\fP is not restricted when testing a filter file
 with \fB\-bf\fP or when testing or verifying addresses using the \fB\-bt\fP or
@@ -1315,12 +1315,12 @@ other circumstances, they are ignored un
 The \fB\-oMa\fP option sets the sender host address. This may include a port
 number at the end, after a full stop (period). For example:
 .sp
-  exim \-bs \-oMa 10.9.8.7.1234
+  exim4 \-bs \-oMa 10.9.8.7.1234
 .sp
 An alternative syntax is to enclose the IP address in square brackets,
 followed by a colon and the port number:
 .sp
-  exim \-bs \-oMa [10.9.8.7]:1234
+  exim4 \-bs \-oMa [10.9.8.7]:1234
 .sp
 The IP address is placed in the \fI$sender_host_address\fP variable, and the
 port, if present, in \fI$sender_host_port\fP. If both \fB\-oMa\fP and \fB\-bh\fP
@@ -1526,22 +1526,22 @@ If other commandline options specify an
 will specify a queue to operate on.
 For example:
 .sp
-  exim \-bp \-qGquarantine
+  exim4 \-bp \-qGquarantine
   mailq \-qGquarantine
-  exim \-qGoffpeak \-Rf @special.domain.example
+  exim4 \-qGoffpeak \-Rf @special.domain.example
 .TP 10
 \fB\-q\fP<\fIqflags\fP> <\fIstart id\fP> <\fIend id\fP>
 When scanning the queue, Exim can be made to skip over messages whose ids are
 lexically less than a given value by following the \fB\-q\fP option with a
 starting message id. For example:
 .sp
-  exim \-q 0t5C6f\-0000c8\-00
+  exim4 \-q 0t5C6f\-0000c8\-00
 .sp
 Messages that arrived earlier than 0t5C6f\-0000c8\-00 are not inspected. If a
 second message id is given, messages whose ids are lexically greater than it
 are also skipped. If the same id is given twice, for example,
 .sp
-  exim \-q 0t5C6f\-0000c8\-00 0t5C6f\-0000c8\-00
+  exim4 \-q 0t5C6f\-0000c8\-00 0t5C6f\-0000c8\-00
 .sp
 just one delivery process is started, for that message. This differs from
 \fB\-M\fP in that retry data is respected, and it also differs from \fB\-Mc\fP in
@@ -1557,7 +1557,7 @@ starting a queue runner process at inter
 single daemon process handles both functions. A common way of starting up a
 combined daemon at system boot time is to use a command such as
 .sp
-  /usr/exim/bin/exim \-bd \-q30m
+  /usr/sbin/exim4 \-bd \-q30m
 .sp
 Such a daemon listens for incoming SMTP calls, and also starts a queue runner
 process every 30 minutes.
@@ -1588,7 +1588,7 @@ regular expression; otherwise it is a li
 If you want to do periodic queue runs for messages with specific recipients,
 you can combine \fB\-R\fP with \fB\-q\fP and a time value. For example:
 .sp
-  exim \-q25m \-R @special.domain.example
+  exim4 \-q25m \-R @special.domain.example
 .sp
 This example does a queue run for messages with recipients in the given domain
 every 25 minutes. Any additional flags that are specified with \fB\-q\fP are
@@ -1704,6 +1704,26 @@ under most shells.
 .sp
 .
 .SH "SEE ALSO"
+.BR exicyclog (8),
+.BR exigrep (8),
+.BR exim_checkaccess (8),
+.BR exim_convert4r4 (8),
+.BR exim_db (8),
+.BR exim_dbmbuild (8),
+.BR exim_lock (8),
+.BR eximon (8),
+.BR exinext (8),
+.BR exiqgrep (8),
+.BR exiqsumm (8),
+.BR exiwhat (8),
+.BR update\-exim4.conf (8),
+.BR update\-exim4defaults (8),
+/usr/share/doc/exim4\-base/,
+/usr/share/doc/exim4\-base/README.Debian.[gz|html].
 .rs
 .sp
 The full Exim specification, the Exim book, and the Exim wiki.
+
+.SH AUTHOR
+This manual page was provided with the upstream Exim source package.
+It was enhanced for the Debian GNU/Linux system.
